NSTTutorials/RuntimeArguments/README.md

```diff
@@ -0,0 +1,61 @@
+[_Return to Tutorials Overview_](../README.md)
+
+# Runtime Arguments
+
+### Overview
+
+NST has several runtime arguments that are supported by default. In this section we will review these runtime arguments, how to add your own custom runtime arguments, and how to add your own custom logger / format writer files.
+
+To see a list of the NST runtime arguments supported out of the box, please see [here](../../NST/README.md#Runtime Arguments).
+
+### Topics Covered
+
+1. [Runtime argument usage](#runtime-argument-usage)
+2. [Adding custom runtime arguments](#adding-custom-runtime-arguments)
+3. [Overriding runtime arguments](#overriding-runtime-arguments)
+
+### References
+- [RuntimeConfigManager](../../NST/src/main/java/com/ebay/runtime/RuntimeConfigManager.java)
+- [TestNG SureFire plugin](https://maven.apache.org/surefire/maven-surefire-plugin/examples/testng.html)
+
+## Runtime argument usage
+
+NST TestNG runtime arguments are specified for NST either as command line arguments, specified in the TestNG Surefire plugin configuration, or optionally as parameters in a TestNG test suite file.
```

Comment:
Include a link to the root readme where all of the runtime arguments are defined.

Reply:
Done in 3d43f0926bdcdf8d1cddbe7d5466662922cf51bb